tutorials/throttle/src/main/java/com/datatorrent/examples/throttle/ThrottlingStatsListener.java

```diff
@@ -26,8 +26,13 @@
     // Slowdown input if the window difference between operators increases beyond this value
     long maxThreshold = 100;
     // restore input operator to normal speed if the window difference falls below this threshold
-    long minThreshold = 100;;
-    boolean normalState = true;
+    long minThreshold = 100;
+
+    Map<Integer, ThrottleState> throttleStates = Maps.newHashMap();
```

Comment:
In that case, suggest adding a comment within that class saying something like: "Can add additional bookkeeping fields like a timestamp here as needed."